File: problems/001/p001.go
package main

import "fmt"

func main() {

	var result int = resolve(3, 5, 1000)

	fmt.Printf("Result=%d\n", result)

}

func resolve(n1 int, n2 int, max int) int {

	var sum int

	for i := 1; i < max; i++ {

		if i%n1 == 0 {
			sum += i
		} else if i%n2 == 0 {
			sum += i
		}
	}

	return sum
}

File: problems/001/p001.js
function main() {

    var result = resolve(3, 5, 1000)

    console.log("Result=%d\n", result)

}

function resolve(n1, n2, max) {

    var sum = 0;

    for (let i = 1; i < max; i++) {

        if (i % n1 == 0) {
            sum += i
        } else if (i % n2 == 0) {
            sum += i
        }
    }

    return sum
}

main()